examples/no_trusted_accounts.tf

```diff
@@ -0,0 +1,22 @@
+  module "terraform_state" {
+    source  = "StratusGrid/terraform-state-s3-bucket-centralized-with-roles/aws"
+    version = "2.0.0"
+    # source  = "github.com/StratusGrid/terraform-aws-terraform-state-s3-bucket-centralized-with-roles"
```

Comment:
This needs to be removed since it's in the TF registry

Reply:
Check now